lib/GithubHandler.js

```diff
@@ -177,15 +177,15 @@ GithubHandler.prototype.pullRequestHandler = function(event, cb) {
     type: 'pull_request', // also in event.event
     service: 'github',
     host: undefined,
+    branch: event.payload.pull_request.head.ref,
```

Comment:
Yeah, I agree. I think the way you have it is good.

The question will be whether we can generate the merge download with a hash and detination instead of a PR number.  But that'll wait for another PR.